src/sentry/incidents/logic.py

```diff
@@ -618,9 +611,20 @@ def create_alert_rule(
             resolve_threshold=resolve_threshold,
             threshold_period=threshold_period,
         )
-        AlertRuleQuerySubscription.objects.create(
-            query_subscription=query_subscription, alert_rule=alert_rule
+        subscriptions = bulk_create_snuba_subscriptions(
```

Comment:
If this transaction fails due to races, or other database problems should we attempt to clean up the subscriptions in snuba?

Reply:
We'll handle all this retry/cleanup logic in the tasks. For now I don't care too much if things end up in a bad state (Snuba isn't even writing data to kafka at this point)